uportal-war/src/main/webapp/WEB-INF/flows/entity-selector/selectEntities.jsp

```diff
@@ -315,6 +315,19 @@
          * Ad Hoc Groups (jsTree)
          */
 
+        $("#groupName").on('input', function () {
+            var nameRegex = /^[\w ]{5,500}$/;
+            if (nameRegex.test($(this).val())) {
+                $("#${n}saveAdHocButton").removeAttr('disabled');
+                $(this).removeClass("invalid");
+            } else {
+                $("#${n}saveAdHocButton").attr('disabled', 'disabled');
+                $(this).addClass("invalid");
```

Comment:
Instead of defining the custom CSS class "invalid" (and associated CSS rules for it), why not use Bootstrap's `.has-error` class:  http://getbootstrap.com/css/#forms-control-validation

This approach would tie into (we hope) the rest of the portal's visual clues for web forms, and can be themed.